projects/ad2s1210_iio/app/ad2s1210_iio.h

```diff
@@ -0,0 +1,37 @@
+/***************************************************************************//**
+* @file   ad2s1210_iio.h
+* @brief  Header file for AD2S1210 IIO interface
+********************************************************************************
+* Copyright (c) 2023 Analog Devices, Inc.
+* Copyright (c) 2023 BayLibre, SAS.
+* All rights reserved.
+*
+* This software is proprietary to Analog Devices, Inc. and its licensors.
+* By using this software you agree to the terms of the associated
+* Analog Devices Software License Agreement.
+*******************************************************************************/
+#ifndef _AD2S1210_IIO_H_
+#define _AD2S1210_IIO_H_
+
+/******************************************************************************/
+/***************************** Include Files **********************************/
+/******************************************************************************/
+
+#include "iio.h"
+#include "iio_types.h"
+#include "ad2s1210.h"
```

Comment:
Are these includes needed? 
If they are required in the source file, should we just move them there?

Reply:
will remove.